engines/csvengine.py

```diff
@@ -86,17 +86,16 @@ def format_insert_value(self, value, datatype):
 
     def insert_statement(self, values):
         """Returns a comma delimited row of values"""
+
         if not hasattr(self, 'auto_column_number'):
             self.auto_column_number = 1
-        offset = 0
-        for i in range(len(self.table.columns)):
-            column = self.table.columns[i]
-            if 'auto' in column[1][0]:
-                values = values[:i + offset] + \
-                    [self.auto_column_number] + values[i + offset:]
-                self.auto_column_number += 1
-                offset += 1
-        return ','.join([str(value) for value in values])
+
+        insert_stmt = ','.join([str(value) for value in values])
+
```

Comment:
I'd probably remove this blank line to make it clearer that this is directly related to the following `if` statement.